src/Round.js

```diff
@@ -18,7 +18,9 @@ class Round {
   }
 
   startRound() {
-    let puzzleClass = new Puzzle(data);
+    let puzzleClass = new Puzzle(data.puzzles);
+    puzzleClass.choosePuzzleBank();
+    puzzleClass.setPuzzle();
     this.puzzle = puzzleClass.newPuzzle;
```

Comment:
I think we might be doing a bit too much setup for this puzzle here. You are:

1. instantiating a new puzzle
2. calling `choosePuzzleBank` on it
3. calling `setPuzzle` on it
4. assigning `this.puzzle` to a `newPuzzle` property

I'd like you to refactor your puzzle class so that this chunk of code can be narrowed down to something like this:

`this.puzzle = new Puzzle(data.puzzles)`

This might mean moving the invocation of `choosePuzzleBank` and `setPuzzle` directly into your constructor method of your puzzle class.